test/reserve/GoodReserveCDai.cap.test.ts

```diff
@@ -140,7 +140,7 @@ describe("GoodReserve - Enforce token cap", () => {
     await expect(
       goodReserve
         .connect(founder)
-        .mintFromReserveRatio(cDai, founder.address, 10)
+        .mintRewardFromRR( founder.address, 10)
```

Comment:
why did the signature change? ie without cdai

Reply:
Since reserve only has cDAI I just put cDAI address from nameservice in function but if you would prefer put address as parameter I can replace

Reply:
yeah lets keep the signatures the same.
we might have multiple reserve tokens in the future